### PR TITLE
Add pow and sqrt function, and add tests.

### PR DESCRIPTION
**Pull request by ChunxiangYu**
Add pow and sqrt.
Add pow sqrt conversion test.

### CODE REVIEW

File: .idea/compiler.xml [hunk 1]
```diff
@@ -6,6 +6,7 @@
         <sourceOutputDir name="target/generated-sources/annotations" />
         <sourceTestOutputDir name="target/generated-test-sources/test-annotations" />
         <outputRelativeToContentRoot value="true" />
+        <module name="Binary-Calculatoryu3" />
```
Comment (Crysis73) [inline .idea/compiler.xml]: Revert this file, we won't want to change the project name

File: .idea/misc.xml [hunk 2]
```diff
@@ -8,7 +8,7 @@
       </list>
     </option>
   </component>
-  <component name="ProjectRootManager" version="2" project-jdk-name="14" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" project-jdk-name="13" project-jdk-type="JavaSDK">
```
Comment (Crysis73) [inline .idea/misc.xml]: We also won't want to change the jdk. You'll want to download jdk 14

File: src/main/java/BinaryOperations.java [hunk 3]
```diff
@@ -54,13 +54,49 @@ public static int Subtract(int A, int B) {
         return difference;
     }
 
+    public static String sqrt(int firstNumber){
+        result = String.valueOf(Math.sqrt(BinaryToDecimal(firstNumber)));
+        return result;
+    }
```
Comment (Crysis73) [inline src/main/java/BinaryOperations.java]: Have this function return an int to be consistent with the rest

File: src/main/java/BinaryOperations.java [hunk 4]
```diff
@@ -54,13 +54,49 @@ public static int Subtract(int A, int B) {
...
 
+    public static String sqrt(int firstNumber){
+        result = String.valueOf(Math.sqrt(BinaryToDecimal(firstNumber)));
+        return result;
+    }
+
+    public static String pow(int firstNumber){
+        result = String.valueOf(Math.pow(BinaryToDecimal(firstNumber),2));
+        return result;
+    }
```
Comment (Crysis73) [inline src/main/java/BinaryOperations.java]: Have this function return an int to be consistent with the rest

**Review by Crysis73 — CHANGES_REQUESTED** [review]

**Review by Jaden-Drury — CHANGES_REQUESTED** [review]
I approve under the condition the suggested changes @Crysis73 made are done.

**Review by Crysis73 — APPROVED** [review]
Approved.

**Pull request merged**